src/Commands/ImportDump.php

```diff
@@ -289,7 +289,12 @@ public function importDump(string $importFilePath, ?bool $optionForce): void
     {
         if ($importFilePath && ($optionForce || $this->confirm(sprintf('Are you sure that you want to import the dump into the database: %s?', $this->protector->getDatabaseName())))) {
             try {
-                $this->protector->importDump($importFilePath, $this->options());
+                $this->protector->importDump($importFilePath, Arr::except($this->options(), ['migrate']));
+
+                if ($this->option('migrate')) {
```

Comment:
this raises the question if the protector->importDump() method needs to be able to perform migrations at all

Reply:
The importDump method must be able to perform migrations if the Protector is implemented in such a way that applications that do not go through the shell and call the importDump method directly can still migrate.